deployment/azuredeploy.json

```diff
@@ -411,6 +454,51 @@
                                 "get"
                             ]
                         }
+                    },

```

Comment:
Now, we seem to be setting the same access control for all the Function Apps. Is there a better way to take care of this instead of duplicating the same sub-object for all the functions? Maybe there's something similar to below usefd for function App configuration.

```
name: [concat(variables('functionList')[copyIndex('FunctionatIndex')], variables('ProjectNameSuffix'))]
```

Reply:
I did my research on this, we can't use the above suggested way, as the `copy` element is used to create multiple instances whereas we want only want a single keyVault